src/op_mode/vrrp.py

```diff
@@ -35,7 +36,15 @@
 
 args = parser.parse_args()
 
+def is_configured():
+    """ Check if VRRP is configured """
+    config = ConfigTreeQuery()
+    if not config.exists(['high-availability', 'vrrp', 'group']):
+        print('VRRP not configured!')
+        exit(1)
```

Comment:
Helper function should return True/False. The call to exit should happen outside of the helper.

Reply:
Done